File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegRental.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegRental
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAORental.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( rental as ClRental) as Boolean
                        Return DAO.DAORental.SelectOne(rental)     
                    End Function
                    Public  Shared Function crear( rental as ClRental) as Integer
                        Return DAO.DAORental.SqlInsert(rental)     
                    End Function
                    Public  Shared Function eliminar( rental as ClRental) as Integer
                        Return DAO.DAORental.SqlDelete(rental)     
                    End Function
                    Public  Shared Function actualizar( rental as ClRental) as Integer
                        Return DAO.DAORental.SqlUpdate(rental)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegRental.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegRental
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAORental.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClRental rental) 
        {
            return DAO.DAORental.sqlSelect(rental);
        }
        public  static int crear(ClRental rental)
        {
            return DAO.DAORental.sqlInsert(rental);
        }
        public  static int eliminar(ClRental rental)
        {
            return DAO.DAORental.sqlDelete(rental);
        }
        public  static int actualizar(ClRental rental)
        {
            return DAO.DAORental.sqlUpdate(rental);
        }
    }
}
